.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -1,27 +1,35 @@
 name: 🐛 Bug report
 description: Submit a bug report to help us make Robocorp better
-title: "[Issue]: "
+labels: [ bug ]
```

Comment:
If this really works, then such label should be automatically applied when a new issue is created with such template.